network-api/networkapi/buyersguide/views.py

```diff
@@ -117,12 +117,27 @@ def product_view(request, slug):
     if product.draft and not request.user.is_authenticated:
         raise Http404("Product does not exist")
 
+    prodcut_dict = product.to_dict()
+    criterions = ['uses_encryption', 'security_updates', 'strong_password',
```

Comment:
language nit: the real plural is `criteria` (american has more and more adopted `criterions`, but they also still use `criteria`, so I'd love to stick with the original plural =)

Reply:
mostly a styling issue: if we're splitting up the array, let's use
```
citeria = [
  'uses_encryption',
  'security_updates',
  'strong_password',
  'manage_vulnerabilities',
  'privacy_policy',
]
```
(also note the comma at the end there: mostly there because that's a python convention, and does not affect `len()`. It'll still see a list of 5 items)